microprot/scripts/process_input.py

```diff
@@ -0,0 +1,66 @@
+import numpy as np
+import click
+import os
```

Comment:
According to PEP8, built-in packages like `os` should appear before 3rd party ones like `click`.